addons/account/models/chart_template.py

```diff
@@ -604,16 +604,16 @@ def _load_template(self, company, code_digits=None, account_ref=None, taxes_ref=
         self.generate_account_groups(company)
 
         # writing account values after creation of accounts
-        for key, value in generated_tax_res['account_dict']['account.tax'].items():
+        for tax, value in generated_tax_res['account_dict']['account.tax'].items():
             if value['cash_basis_transition_account_id']:
-                AccountTaxObj.browse(key).write({
+                tax.write({
```

Comment:
`tax.cash_basis_transition_account_id = ...`
Also, are you sure `value['cash_basis_transition_account_id']` is not returning a record then?